rai/lib/interface.cpp

```diff
@@ -37,24 +37,21 @@ int xrb_uint256_from_string (const char * source, xrb_uint256 destination)
 {
 	auto & number (*reinterpret_cast<rai::uint256_union *> (destination));
 	auto error (number.decode_hex (source));
-	auto result (error ? 1 : 0);
-	return result;
+	return error ? 1 : 0;
 }
 
 int xrb_uint512_from_string (const char * source, xrb_uint512 destination)
 {
 	auto & number (*reinterpret_cast<rai::uint512_union *> (destination));
 	auto error (number.decode_hex (source));
-	auto result (error ? 1 : 0);
-	return result;
+	return error ? 1 : 0;
```

Comment:
Do these match the polarity of the stdlib `EXIT_SUCCESS` / `EXIT_FAILURE`  defines ?  Can we use those?

Reply:
Those are usually only used for std::exit. 

If we were to use them, we need to be careful since EXIT_SUCCESS is not required by the standard to be 0. It basically excludes the `if (!something())` idiom.